model/riscv_insts_vext_arith.sail

```diff
@@ -0,0 +1,531 @@
+/* ******************************************************************************* */
+/* This file implements part of the vector extension.                              */
+/* Chapter 11: vector integer arithmetic instructions                              */
+/* Chapter 12: vector fixed-point arithmetic instructions                          */
+/* Chapter 16: vector permutation instructions                                     */
+
+/* ******************************************************************************* */
+
+/* **************************OPIVV(VVTYPE)**************************************** */
```

Comment:
Spaces?

Reply:
The format of headings are all modified.